### ISSUE TITLE
ModuleNotFoundError after upgrading to Django 3.0

### ISSUE THREAD

**Issue by jar3b** [PokaInc/django-admin-hstore-widget#14]
Version `1.0.1` from pypi is incompatible with Django 3.0, the follwing error was produced after `runserver`:

```
Traceback (most recent call last):
  File "/usr/lib/python3.7/threading.py", line 926, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.7/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/jar3b/work/test-project/venv/lib/python3.7/site-packages/django/utils/autoreload.py", line 53, in wrapper
    fn(*args, **kwargs)
  File "/home/jar3b/work/test-project/venv/lib/python3.7/site-packages/django/core/management/commands/runserver.py", line 109, in inner_run
    autoreload.raise_last_exception()
  File "/home/jar3b/work/test-project/venv/lib/python3.7/site-packages/django/utils/autoreload.py", line 76, in raise_last_exception
    raise _exception[1]
  File "/home/jar3b/work/test-project/venv/lib/python3.7/site-packages/django/core/management/__init__.py", line 357, in execute
    autoreload.check_errors(django.setup)()
  File "/home/jar3b/work/test-project/venv/lib/python3.7/site-packages/django/utils/autoreload.py", line 53, in wrapper
    fn(*args, **kwargs)
  File "/home/jar3b/work/test-project/venv/lib/python3.7/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/jar3b/work/test-project/venv/lib/python3.7/site-packages/django/apps/registry.py", line 122, in populate
    app_config.ready()
  File "/home/jar3b/work/test-project/venv/lib/python3.7/site-packages/django/contrib/admin/apps.py", line 24, in ready
    self.module.autodiscover()
  File "/home/jar3b/work/test-project/venv/lib/python3.7/site-packages/django/contrib/admin/__init__.py", line 26, in autodiscover
    autodiscover_modules('admin', register_to=site)
  File "/home/jar3b/work/test-project/venv/lib/python3.7/site-packages/django/utils/module_loading.py", line 47, in autodiscover_modules
    import_module('%s.%s' % (app_config.name, module_to_search))
  File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/jar3b/work/test-project/my_app/admin.py", line 3, in <module>
    from django_admin_hstore_widget.forms import HStoreFormField
  File "/home/jar3b/work/test-project/venv/lib/python3.7/site-packages/django_admin_hstore_widget/forms.py", line 4, in <module>
    from .widgets import HStoreFormWidget
  File "/home/jar3b/work/test-project/venv/lib/python3.7/site-packages/django_admin_hstore_widget/widgets.py", line 2, in <module>
    from django.contrib.admin.templatetags.admin_static import static
ModuleNotFoundError: No module named 'django.contrib.admin.templatetags.admin_static'
```

The reason is because django marked `django.contrib.admin.templatetags.admin_static` package  as deprecated since version 2.1 and dropped their support in 3.0.

Solutions are:
1. Drop support for old django versions (idk which, pre-2.1?) and rewrite import to `django.templatetags.static`
2. Use "imports failback" or django version detection.

**Comment by alexandredufour** [comment]
Should be fixed in version 1.1.0 ( released today )

**Comment by sheddyowden** [comment]
could someone provide the solution to that please ?? am confussed i cant django server
